Origin: https://sources.debian.org/src/iproute2/6.15.0-1/debian/patches/0001-Add-moo-feature.patch/
Source-package: iproute2_6.15.0-1.debian.tar.xz (SHA256 780375193dffeb85f310a918b806f2f4e480f76b815a1f0f1eac5d0488ea1a8f)

Author: Alexander Wirt <formorer@debian.org>
Description: Add moo feature
 critical value-add business feature for Debian
Forwarded: not-needed
--- a/ip/ip.c
+++ b/ip/ip.c
@@ -87,6 +87,19 @@
 	return 0;
 }
 
+static int do_moo(int argc, char **argv)
+{
+	fprintf(stderr,
+"\n"
+" _ __ ___   ___   ___\n"
+"| '_ ` _ \\ / _ \\ / _ \\\n"
+"| | | | | | (_) | (_) |\n"
+"|_| |_| |_|\\___/ \\___/\n"
+"\n\n"
+"P.S. no real cows were harmed for this moo\n");
+	exit(1);
+}
+
 static const struct cmd {
 	const char *cmd;
 	int (*func)(int argc, char **argv);
@@ -125,6 +138,7 @@
 	{ "ioam",	do_ioam6 },
 	{ "help",	do_help },
 	{ "stats",	do_ipstats },
+	{ "moo",	do_moo },
 	{ 0 }
 };
 
